src/engraving/libmscore/score.cpp

```diff
@@ -2738,6 +2742,10 @@ void Score::removeStaff(Staff* staff)
     mu::remove(_staves, staff);
     staff->part()->removeStaff(staff);
 
+    if (staff->isSystemObjectStaff()) {
```

Comment:
isSystemObjectStaff() should be declared inside Score. This code looks strange now